tests/hypervisor/test_esx.py

```diff
@@ -157,6 +159,41 @@ def test_exclude_hosts(self, virtwho, function_hypervisor, hypervisor_data):
                     and result['thread'] == 1
                     and hypervisor_id_data not in str(result['mappings']))
 
+    @pytest.mark.tier1
+    def test_fake_type(self, virtwho, function_hypervisor, hypervisor_data):
```

Comment:
We should also test the host-guest association in mapping and web, and the vdc pool's subscription.
So maybe we can new create a tests/hypervisor/fake.py to cover the case in each hypervisor modes.
What's your opinion?

Reply:
As discussed, we will consider this in the future, have added the `#TODO` note.